app/frontend/src/authConfig.ts

```diff
@@ -88,18 +89,24 @@ export const getRedirectUri = () => {
 // Get an access token if a user logged in using app services authentication
 // Returns null if the app doesn't support app services authentication
 const getAppServicesToken = (): Promise<AppServicesToken | null> => {
-    return fetch(appServicesAuthTokenUrl).then(r => {
+    return fetch(appServicesAuthTokenRefreshUrl).then(r => {
```

Comment:
So this is constantly refreshing? Is that the general recommendation? (Versus refreshing only based on expiration or some such?) Might be a naive question

Reply:
There doesn't seem to be a solid recommendation on how often to do this

https://learn.microsoft.com/en-us/azure/app-service/configure-authentication-oauth-tokens#refresh-auth-tokens

Reply:
Okay, CC @jamesc in case he has any thoughts, but this seems fine to ship.